Gemfile.lock

```diff
@@ -177,7 +177,7 @@ GEM
       railties (>= 3.0.0)
     faker (1.7.3)
       i18n (~> 0.5)
-    ffi (1.11.3)
+    ffi (1.12.2)
```

Comment:
### [ffi](https://github.com/ffi/ffi/wiki)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.11.3 → 1.12.2

[_~~change-log~~, [source-code](http://github.com/ffi/ffi)_]





<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/ffi/ffi/compare/1.11.3...1.12.2).

These are the first 10 commits:
- (e908e0e) [Update to rake-compiler-dock-1.0.0 to build ruby-2.7 binary …](https://github.com/ffi/ffi/commit/e908e0e6d3c574efe3ed9f05bdad09d09004911d)
- (7896c53) [Update rake to avoid warnings on ruby-2.7](https://github.com/ffi/ffi/commit/7896c530240dcb13f88fadede1150fe4d1db3929)
- (5694fd8) [Set ruby-2.3.8 to allow_failures because of ffi unrelated er…](https://github.com/ffi/ffi/commit/5694fd8951043dda6e8862ac34ba230e9556eb83)
- (26a22ba) [Require "ffi/version" per default](https://github.com/ffi/ffi/commit/26a22baddc2cdb0b37af5dca2d07e54524ca1bae)
- (44008db) [Disallow struct layout changes](https://github.com/ffi/ffi/commit/44008dbc422d34e73cbb739b7ed701c3d9d599c6)
- (bd45ff5) [Namespace all benchmarks since they are loaded into one proc…](https://github.com/ffi/ffi/commit/bd45ff5913757d67d96d81f24ef7a919dbdf25ce)
- (89f4258) [Warn about Struct layout redefinition only](https://github.com/ffi/ffi/commit/89f4258ca9d637bba8f0d149a0c764aff5ff864d)
- (d564eb4) [Merge pull request #735 from larskanis/disallow-re-layout](https://github.com/ffi/ffi/commit/d564eb451a6372d460b156f3269381701f79ec97)
- (da4402f) [Update libffi to latest master branch](https://github.com/ffi/ffi/commit/da4402fd007b03b440c69f55c3e1fd8222294f09)
- (62c0c97) [Update CHANGELOG for version 1.12.0](https://github.com/ffi/ffi/commit/62c0c9712acb0b94c15cd22d316149e71bcf9658)


</details>